pygmtools/classic_solvers.py

```diff
@@ -1061,6 +1062,188 @@ def ipfp(K, n1=None, n2=None, n1max=None, n2max=None, x0=None,
         return result
 
 
+def a_star(feat1, feat2, A1, A2, n1=None, n2=None,network=None, 
+           return_network=False, pretrain='AIDS700nef',backend=None,**kwargs):
+
+    r"""
+    GENN-A* solver for graph matching  based on Graph Neural Network.
```

Comment:
Add more descriptions for GENN, A-star algorithm, graph edit distance problem, citation information, etc.